geometry_msgs/Pose pick_pose
